awx/main/models/workflow.py

```diff
@@ -588,6 +616,12 @@ def node_prompts_rejected(self):
                 node_list.append(node.pk)
         return node_list
 
+    def save(self, *args, **kwargs):
+
+        self.validate_unique()
+
+        super(WorkflowJobTemplate, self).save(*args, **kwargs)
```

Comment:
This `validate_unique` is only used to enforce `SOFT_UNIQUE_TOGETHER`, and it's in the base model. I really don't think this should be necessary. I can't think of a good reason we shouldn't delete this `save` method definition here.